### PR TITLE
Add splatter brush 

### PR DESCRIPTION
**Pull request by me4502**
Continuation of https://github.com/EngineHub/WorldEdit/pull/1547

Uses a slightly different algorithm (and therefore different output, this has more of a dropoff effect with high decay rather than splotches), but moves the concept into a mask for more WorldEdit-y usage

### CODE REVIEW

File: worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java [hunk 1]
```diff
@@ -134,6 +134,7 @@
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.Random;
```
Comment (octylFractal) [inline worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java:137]: why

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java [hunk 2]
```diff
@@ -175,6 +176,36 @@ public void cylinderBrush(Player player, LocalSession session,
...
+            return;
+        }
+
+        BrushTool tool = session.getBrushTool(player.getItemInHand(HandSide.MAIN_HAND).getType());
+        tool.setFill(pattern);
+        tool.setSize(radius);
+
+        tool.setBrush(new SplatterBrush(decay), "worldedit.brush.splatter");
+
+        player.printInfo(TranslatableComponent.of("worldedit.brush.splatter.equip", TextComponent.of((int) radius), TextComponent.of(decay)));
```
Comment (octylFractal) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java:205]: Why aren't we passing down the double radius directly?

File: worldedit-core/src/main/java/com/sk89q/worldedit/function/mask/SplatterMask.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+        double decayChance = distRatio * decay * 2;
+
+        return ThreadLocalRandom.current().nextDouble() > decayChance;
+    }
+
+    @Nullable
+    @Override
+    public Mask2D toMask2D() {
+        return null;
```
Comment (octylFractal) [inline worldedit-core/src/main/java/com/sk89q/worldedit/function/mask/SplatterMask.java:57]: Please do implement a Mask2D variant.

**Review by octylFractal — CHANGES_REQUESTED** [review]

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java (hunk 2 above)
Comment (me4502) [reply worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java:205]: All of them do this - I'd rather keep this as-is for now.

**Review by octylFractal — APPROVED** [review]

**Pull request merged**